easybuild/easyconfigs/f/FSL/FSL-5.0.10_makefile_fixes.patch

```diff
@@ -0,0 +1,276 @@
+# This fixes the build system of FSL. It now (mostly) uses EB.
+# Ward Poelmans <wpoely86@gmail.com>
+diff -ur fsl.orig/config/common/vars.mk fsl/config/common/vars.mk
+--- fsl.orig/config/common/vars.mk	2016-05-24 17:44:11.000000000 +0200
++++ fsl/config/common/vars.mk	2017-05-23 22:14:37.138797842 +0200
+@@ -24,14 +24,14 @@
+ USRCFLAGS =
+ USRCXXFLAGS =
+
+-LDFLAGS = ${ARCHLDFLAGS} ${USRLDFLAGS} -L. -L${DEVLIBDIR} -L${LIBDIR}
++LDFLAGS = ${EBVARLDFLAGS} ${ARCHLDFLAGS} ${USRLDFLAGS} -L. -L${DEVLIBDIR} -L${LIBDIR}
+
+-AccumulatedIncFlags = -I${INC_BOOST} ${USRINCFLAGS} -I. -I${DEVINCDIR} -I${INCDIR}
++AccumulatedIncFlags = ${EBVARCPPFLAGS} -I${INC_BOOST} ${USRINCFLAGS} -I. -I${DEVINCDIR} -I${INCDIR}
+
+-CFLAGS = ${ANSI_FLAGS} ${ANSI_CFLAGS} ${DBGFLAGS} ${USEDCSTATICFLAGS} ${USRCFLAGS} ${ARCHFLAGS} ${OPTFLAGS}  \
++CFLAGS = ${EBVARCFLAGS} ${ANSI_FLAGS} ${ANSI_CFLAGS} ${DBGFLAGS} ${USEDCSTATICFLAGS} ${USRCFLAGS} ${ARCHFLAGS} ${OPTFLAGS}  \
+ 	${AccumulatedIncFlags}
+
+-CXXFLAGS = ${ANSI_FLAGS} ${ANSI_CXXFLAGS} ${DBGFLAGS} ${USEDCXXSTATICFLAGS} ${USRCXXFLAGS} ${ARCHFLAGS} ${OPTFLAGS}  \
++CXXFLAGS = ${EBVARCXXFLAGS} ${ANSI_FLAGS} ${ANSI_CXXFLAGS} ${DBGFLAGS} ${USEDCXXSTATICFLAGS} ${USRCXXFLAGS} ${ARCHFLAGS} ${OPTFLAGS}  \
+ 	${AccumulatedIncFlags}
+
+ HFILES = *.h
+Only in fsl/config/common: vars.mk.orig
+diff -ur fsl.orig/config/generic/systemvars.mk fsl/config/generic/systemvars.mk
+--- fsl.orig/config/generic/systemvars.mk	2007-07-13 13:00:20.000000000 +0200
++++ fsl/config/generic/systemvars.mk	2017-05-23 22:14:37.139797850 +0200
+@@ -16,8 +16,8 @@
+
+ # Compiler dependent variables
+
+-CC = gcc
+-CXX = c++
++CC := ${CC}
++CXX := ${CXX}
+ CSTATICFLAGS = -static
+ CXXSTATICFLAGS = -static
+
+@@ -25,7 +25,7 @@
+
+ DEPENDFLAGS = -MM
+
+-OPTFLAGS =  -O3 -fexpensive-optimizations ${ARCHFLAGS}
++OPTFLAGS := ${CFLAGS} ${ARCHFLAGS} -wd803
```

Comment:
better add `-wd803` via the easyconfig to avoid making this patch specific to Intel compilers?